specifications/api/swagger.yaml

```diff
@@ -320,32 +320,16 @@ x-stakePoolId: &stakePoolId
   <<: *addressId
   description: A unique identifier (i.e. its public key)
 
-x-stakePoolTicker: &stakePoolTicker
-  type: string
-  minLength: 3
-  maxLength: 4
-  description: A short human-readable identifier
-  example: IOHK
-
 x-stakePoolMetrics: &stakePoolMetrics
   type: object
   required:
     - controlled_stake
-    - performance
-    - last_blocks
+    - produced_blocks
   properties:
     controlled_stake: *transactionAmount
-    performance:
-      description: Estimated performance computed over the last epoch
-      <<: *percentage
-
-    last_blocks:
+    produced_blocks:
       <<: *numberOfBlocks
-      description: Number of blocks correctly processed during the last epoch
-
-x-stakePoolProfitMargin: &stakePoolProfitMargin
-  <<: *percentage
-  description: What proportion of the reward is paid to the pool operator
+      description: Number of blocks correctly processed by a given stake pool
```

Comment:
Perhaps `number of blocks produced ...` ? or `number of blocks created ...` ? I think I wrote the original description, but it doesn't sound quite right.